sdk/identity/azure-identity/test/ut/azure_pipelines_credential_test.cpp

```diff
@@ -644,7 +644,7 @@ TEST(AzurePipelinesCredential, InvalidServiceConnectionId_LIVEONLY_)
   }
 }
 
-TEST(AzurePipelinesCredential, InvalidSystemAccessToken_LIVEONLY_)
+/*TEST(AzurePipelinesCredential, InvalidSystemAccessToken_DISABLED_)
```

Comment:
Rather than commenting out the test, please disable it, by renaming the test like this (prefix with `DISABLED_`):
https://github.com/Azure/azure-sdk-for-cpp/pull/5785

> DISABLED_InvalidSystemAccessToken_LIVEONLY_

See:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3954813&view=logs&j=4548f7fc-05cc-57aa-a896-3f4bc48e7aad&t=a90428ec-b6a1-582b-bdc6-58150785e27b

> 38 - azure-identity.AzurePipelinesCredential.InvalidSystemAccessToken_LIVEONLY_ (Disabled)

Reply:
Please pull from main which contains the changes from https://github.com/Azure/azure-sdk-for-cpp/pull/5785.

That way, you don't have to modify the identity test files in this PR, at all.